extensions/amp-analytics/0.1/test/vendor-requests.json

```diff
@@ -352,6 +352,11 @@
 	  "baseSuffix": "Service=!service&Category=!category&Url=_source_url_&Device=!device&uid=_random_",
 	  "pageview": "https://t5.mindtake.com/tag/cid/!code/track.gif?Service=!service&Category=!category&Url=_source_url_&Device=!device&uid=_random_"
   },
+  "kenshoo": {
+    "host": "https://amp.xg4ken.com/",
+    "parameters": "ampcid=${clientId}&chcid=${channelClickId}&tid=${tid}&uid=${userId}&domain=${canonicalHostname}",
+    "landingPage": "https://amp.xg4ken.com/amp/v1/match?ampcid=!clientId&chcid=!channelClickId&tid=!tid&uid=!userId&domain=!canonicalHostname"
+  },
```

Comment:
Should be: "landingPage": "${host}/amp/v1/match?${parameters}"

Reply:
@ifatgv4 it should be the final output of the url.